### PR TITLE
Forum Page Discover - Created connection with topics API

### PR DESCRIPTION
**Pull request by Bcastillo44**
- Related Issue (include '#'): #670 

- Description of changes made:
     Created connection between topics from API to topics on Forum Page Discover

- Is the feature complete/bug resolved/etc..:

     Need to add pagination

### CODE REVIEW

File: client/src/pages/ForumPage/ForumPageDiscover.js [hunk 1]
```diff
@@ -20,7 +23,29 @@ const handleClick = e => {
...
+  axios.get(`${BASE_URL}/topics`)
+    .then(response => {
+      this.setState({ 
+        topics: response.data
+      })
+    })
+    .catch(error => {
+      console.log('Error', error);
+    })
+}
```
Comment (0xZakk) [inline client/src/pages/ForumPage/ForumPageDiscover.js]: Can you put this in the appropriate service in the `services/` directory?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@Bcastillo44

File: client/src/pages/ForumPage/ForumPageDiscover.js (hunk 1 above)
Comment (Bcastillo44) [reply client/src/pages/ForumPage/ForumPageDiscover.js]: Added Forum.js page in services/ directory and created import/export of axios fetch for Topics

Comment (0xZakk) [inline client/src/pages/ForumPage/ForumPageDiscover.js]: Can you remove this and use the one you created in the service?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@Bcastillo44

File: client/src/pages/ForumPage/ForumPageDiscover.js (hunk 1 above)
Comment (Bcastillo44) [reply client/src/pages/ForumPage/ForumPageDiscover.js]: Not sure why that showed up on your end, double-checked and removed

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@Bcastillo44 Formatting here too (per our slack conversation)

**Review by 0xZakk — CHANGES_REQUESTED** [review]
This is what I'm seeing when I load up the forum topics:

Screen Shot 2020-07-14 at 10 56 40 AM

There aren't any topics?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
I'm getting this now on the Forum topics page: 

Screen Shot 2020-07-14 at 1 56 25 PM

Screen Shot 2020-07-14 at 1 57 00 PM

**Comment by secondaryfun** [comment]
Branch https://github.com/CotripperPlatform/CoTrip/pull/685 has these changes. This branch can be closed without merging.

**Pull request closed without merging**